fieldservice_sign/wizard/sign_send_request.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2019 Open Source Integrators
+# Copyright (C) 2019 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import fields, models, _
+
+
+class SignSendRequest(models.TransientModel):
+    _inherit = 'sign.send.request'
+
+    fsm_order_id = fields.Many2one('fsm.order', 'FSM Order')
+
+    def send_request(self):
+        res = self.create_request()
+        request = self.env['sign.request'].browse(res['id'])
+        request.fsm_order_id = self.fsm_order_id.id
+        return {
```

Comment:
I'm guessuig that `res` is a Client Action dict.
If so, we can simplify this and just `return res`, instead of redefining the Client Action.

Reply:
Sorry, my bad - this is not an inherited method, should be OK as is.